ui/packages/components/src/RunDetailsV2/RunDetails.tsx

```
@@ -82,6 +84,13 @@ export function RunDetails(props: Props) {
     setPollInterval(undefined);
   }
 
+  // Do not show the error if queued and the error is no spans
+  const isNoSpansFoundError = !!runRes.error?.toString().match(/no function run span found/gi);
+  const showError =
```

That's better, my check could have swallowed other valid errors.